cmd/release-controller/http.go

```diff
@@ -615,25 +615,15 @@ func (c *Controller) httpReleaseInfoDownload(w http.ResponseWriter, req *http.Re
 	http.Redirect(w, req, u, http.StatusFound)
 }
 
-func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
-	start := time.Now()
-	defer func() { glog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
-
-	vars := mux.Vars(req)
-	release := vars["release"]
-	tag := vars["tag"]
-	from := req.URL.Query().Get("from")
-
+func (c *Controller) getReleaseTagInfo(release, tag, from string) (*ReleaseStreamTag, string, string, error, int) {
```

Comment:
Smells bad to leak out the HTTP code here -- either interpret the result into a code in `httpReleaseInfo()` or return a type which implements `Error()` and exposes the code?